tensorbay/client/job.py

```diff
@@ -55,11 +55,11 @@ class Job(AttrsMixin, ReprMixin):  # pylint: disable=too-many-instance-attribute
     job_id: str = attr(key=camel)
     arguments: Dict[str, Any] = attr()
     created_at: int = attr(key=camel)
-    started_at: Optional[int] = attr(key=camel)
-    finished_at: Optional[int] = attr(key=camel)
+    started_at: Optional[int] = attr(key=camel, default=None)
```

Comment:
The word `some` is unclear in commit message,  also it is weird that there are two `of` in a sentence, here is my advice:
`feat(client): use AttrMixin to handle the default values for Job attrs`